### DISCUSSION TITLE
Unable to list contents for 'livewire-tmp', deep listing

### DISCUSSION THREAD

**Discussion by ousid** [livewire/livewire#5280] [1. Help]
Hello folks.

The past few days, I released a package called [Laravel CSV](https://github.com/coderflexx/laravel-csv) Using livewire under the hood, and the tests are fine in my local development (Using Linux) But I have faced a few failed tests in GitHub CI, on windows latest,

```bash
Unable to list contents for 'livewire-tmp', deep listing

Reason: SplFileInfo::getMTime(): stat failed for D:\a\laravel-csv\laravel-csv\vendor\orchestra\testbench-core\laravel\storage\framework/testing/disks/tmp-for-tests\livewire-tmp\OpC0hrGObr7G4bu9pUzYCSfE7LbYb6-metaY3VzdG9tZXJzLmNzdg==--size=120120.csv
```

And I'm not sure what's exactly the issue here.

Using `Livewire v2.10` 
Here's an example from the tests:
```php
it('throws a validation error if the csv file empty', function () {
    $model = Customer::class;

    $file = UploadedFile::fake()
        ->createWithContent(
            'customers.csv',
            file_get_contents('stubs/empty.csv', true)
        );

    livewire(CsvImporter::class, [
        'model' => $model,
    ])
    ->set('file', $file)
    ->assertSet('model', $model)
    ->assertHasErrors(['file']);
});
```

If you want more details, please take a look [here](https://github.com/coderflexx/laravel-csv/actions/runs/3169700839/jobs/5161792955)

Thanks in advance!

**Comment by camsimi** [comment]
Hi @ousid, your issue is old but I'm wondering if you had came out with a solution ? I'm in a similar situation with Livewire 3 and Pest, trying to test a Livewire Component with a file uploaded. Tests ok when run from Phpstorm but not from docker container in the CI.

**Reply by ousid** [reply]
Hi, @camsimi 

Unfortunately, I did not find a solution for this issue, and as you can see by going to the repo, the issue still the same. I didn't check it since.

**Reply by camsimi** [reply]
I fixed the issue faking the Storage :)

**Reply by ousid** [reply]
That's perfect! Will try it out in my end, and let you know.

Thanks for pointing this out @camsimi!
